src/Domain/CoordinateMaps/KerrHorizonConforming.cpp

```diff
@@ -10,15 +10,24 @@
 #include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
 #include "Utilities/ContainerHelpers.hpp"
 #include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/ErrorHandling/Assert.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/Gsl.hpp"
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/StdArrayHelpers.hpp"
 
 namespace domain::CoordinateMaps {
 
-KerrHorizonConforming::KerrHorizonConforming(std::array<double, 3> spin)
-    : spin_(spin), spin_mag_sq_(dot(spin, spin)) {}
+KerrHorizonConforming::KerrHorizonConforming(
+    const double mass, const std::array<double, 3> dimensionless_spin)
+    : spin_(mass * dimensionless_spin),
```

Comment:
I would rename `spin_` to `a_` or `kerr_a` or `spin_parameter_`

Reply:
next fixup